js/core/utils.js

```diff
@@ -27,15 +27,48 @@ define(
             return $(arr);
         };
 
-        // For any element, returns a title string that applies the algorithm used for determining the
+        // For any element, returns an array of title strings that applies
+        // the algorithm used for determining the
         // actual title of a <dfn> element (but can apply to other as well).
-        $.fn.dfnTitle = function () {
-            var title;
-            if (this.attr("title")) title = this.attr("title");
+        // 
+        // if isDefinition is true, then the element is a definition, not a 
+        // reference to a definition.  Any @title or @lt will be replaced with
+        // @data-lt to be consistent with Bikeshed / Shepherd.
+        $.fn.dfnTitle = function ( isDefinition ) {
```

Comment:
Also, `dfnTitle` looks like a getter. Maybe give it a slightly more descriptive name?